public/css/icinga/menu.less

```diff
@@ -286,6 +286,8 @@
 
         &:first-of-type {
           padding-left: 1.5em;
+          text-overflow: ellipsis;
+          overflow: hidden;
```

Comment:
There is already a rule for this, albeit not for the collapsed menu. Please re-use it.